test/interactive_markers/test_interactive_marker_server.cpp

```diff
@@ -367,7 +367,7 @@ TEST_F(TestInteractiveMarkerServerWithMarkers, get_interactive_markers_communica
   using namespace std::chrono_literals;
 
   MockInteractiveMarkerClient::SharedFuture future = mock_client_->requestInteractiveMarkers();
-  auto ret = executor_.spin_until_future_complete<MockInteractiveMarkerClient::SharedResponse>(
+  auto ret = executor_.spin_until_future_complete<MockInteractiveMarkerClient::SharedFuture>(
```

Comment:
nit: delete the explicit specialization.
Template argument deduction will do the magic.